src/api/app/helpers/webui/monitor_helper.rb

```diff
@@ -4,4 +4,15 @@ def self.print_statistics_array(array)
     array ||= []
     '[' + array.map { |time, value| "[#{time * 1000}, #{value}]" }.join(',') + ']'
   end
+
+  def icon_for_daemon(state)
```

Comment:
The commit message says you are moving `kiwi_image_breadcrumb` not this.

Reply:
Thanks! Fixed.